packages/components/src/components/button/_button.scss

```diff
@@ -132,10 +132,10 @@
 
     &:hover,
     &:active {
-      color: $ibm-color__blue-70;
+      color: $interactive-04;
 
       .#{$prefix}--btn__icon path {
```

Comment:
This is unrelated to the PR but would we need to open up this selector for other types of svg nodes like `circle`/`rect`/etc?

Reply:
CC @emyarod for his thoughts.

Reply:
yes we can target `svg` instead